### DISCUSSION TITLE
Introducing DB4S Universal Build for macOS (Testing version)

### DISCUSSION THREAD

**Discussion by lucydodo** [sqlitebrowser/sqlitebrowser#3375] [Show and tell]
I am so excited to introduce a testing version of DB Browser for SQLite - the universal build for macOS!
I look forward to providing an even better experience for our valued users with this development.

Please remember that this universal build of DB Browser for SQLite is currently in the testing phase.
and I encourage your feedback to help us refine and improve it further. Thanks. :)

You can download it from [here.](https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/testing)

**Comment by chrisjlocke** [comment]
I'll this as a blog post and Twitter.

**Comment by lucydodo** [comment]
The test is over, and we now serve the universal binary by default for nightly builds.
Universal binaries for stable version will be released starting with the next stable version a bit more testing.

Until we release a new stable version, you can get the nightly builds here:
https://nightlies.sqlitebrowser.org/latest/ or https://nightlies.sqlitebrowser.org/macos-universal/

Thanks.

**Reply by justinclift** [reply]
Awesome! :smile:

The README file in the nightlies latest directory should probably be updated too:

https://nightlies.sqlitebrowser.org/latest/%2A%20README.txt

You ok to get that done?

**Reply by justinclift** [reply]
~Oh, that means I should turn off the macOS intel nightly build cronjob now yeah?~

I've disabled the macOS Intel nightly build job. :smile:

**Reply by lucydodo** [reply]
Oh, there's something I missed. Thanks for letting me know, I just updated it. :)

**Reply by justinclift** [reply]
Cool.  I just tweaked the wording a bit as well, to hopefully make the universal binaries thing super clear. :smile:
